ios/FluentUI/PersonaButton/PersonaButton.swift

```diff
@@ -77,13 +77,17 @@ public struct PersonaButton: View, ConfigurableTokenizedControl {
         }
         .frame(minWidth: adjustedWidth, maxWidth: adjustedWidth, minHeight: 0, maxHeight: .infinity)
         .background(Color(dynamicColor: tokens.backgroundColor))
-        .resolveTokens(self)
     }
 
     @Environment(\.fluentTheme) var fluentTheme: FluentTheme
     @Environment(\.sizeCategory) var sizeCategory: ContentSizeCategory
     @ObservedObject var state: MSFPersonaButtonStateImpl
-    var tokens: PersonaButtonTokens { state.tokens }
+    let defaultTokens: PersonaButtonTokens = .init()
+    var tokens: PersonaButtonTokens {
+        let tokens = resolvedTokens()
+        tokens.size = state.buttonSize
```

Comment:
Related to the comment I made in another file:
`After all these changes is there any benefit on still keeping the override tokens in the state object?`

Since we're propagating the state object's properties here, is there any reason why the override tokens should be in the state object? It could eventually be another property in the SwiftUI control just like the UIkit ones have it?

Reply:
We can't store a mutable object reference in a SwiftUI view because the struct is immutable and prone to recreation. The only way we could keep `overrideTokens` in the view is it if was part of the designated initializer for every control, which doesn't feel quite right to me.

Reply:
Discussed offline: we'll continue to think about how to bring tokens together, but for now we'll merge this PR to unblock other work.